### PR TITLE
Update: JS doc update 

### PR DESCRIPTION
**Pull request by harsha509**
Updated ImplicitWait as per W3C standard mentioned for WebdriverJs

Signed-off-by: Sri Harsha <sri_harsha509@hotmail.com>

### CODE REVIEW

**Comment by diemol** [comment]
No need to close the PR, the Travis build might fail if the GitHub api quota has been reached.

**Review by diemol — CHANGES_REQUESTED** [review]
@Harsha509 Could you please add the code sample to the translated pages as well?

**Comment by harsha509** [comment]
@diemol, Updated the requested changes.

**Comment by harsha509** [comment]
> @Harsha509 Could you please add the code sample to the translated pages as well?

Its done.

**Review by diemol — APPROVED** [review]
Thank you!

File: docs_source_files/content/webdriver/waits.en.md [hunk 1]
```diff
@@ -317,7 +317,17 @@ my_dynamic_element = driver.find_element_by_id("myDynamicElement")
 # We don't have a Ruby code sample yet -  Help us out and raise a PR  
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+(async function(){
+
+// Apply timeout for 10 seconds
+driver.manage().setTimeouts( { implicit: 10000 } );
```
Comment (jleyba) [inline docs_source_files/content/webdriver/waits.en.md:323]: Needs an `await`

File: docs_source_files/content/webdriver/waits.en.md [hunk 2]
```diff
@@ -317,7 +317,17 @@ my_dynamic_element = driver.find_element_by_id("myDynamicElement")
...
+
+// Navigate to url
+driver.get('http://somedomain/url_that_delays_loading');
```
Comment (jleyba) [inline docs_source_files/content/webdriver/waits.en.md:326]: Needs an `await`

**Review by jleyba — COMMENTED** [review]
All of the `driver.manage().setTimeouts()` and `driver.get()` calls need `await`

File: docs_source_files/content/webdriver/waits.en.md (hunk 1 above)
Comment (harsha509) [reply docs_source_files/content/webdriver/waits.en.md:323]: Thanks @jleyba created a new PR with changes https://github.com/SeleniumHQ/docs/pull/344

File: docs_source_files/content/webdriver/waits.en.md (hunk 2 above)
Comment (harsha509) [reply docs_source_files/content/webdriver/waits.en.md:326]: Thanks @jleyba created a new PR with changes https://github.com/SeleniumHQ/docs/pull/344

**Pull request merged**